src/components/pages/ArticlePage/Toegankelijkheid.tsx

```diff
@@ -0,0 +1,112 @@
+import type { FC } from 'react'
+
+import { Heading, Paragraph, List, ListItem, Link } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+import BasePage from '../BasePage'
+
+const StyledHeader = styled(Heading)`
+  font-size: 1.125rem;
+`
+
+const Toegankelijkheid: FC = () => (
+  <BasePage documentTitle="Toegankelijkheid" pageTitle="Toegankelijkheid">
+    <article>
```

Comment:
We can get rid of this as well, the entire thing is already wrapped in an `article`.